src/azure-cli/requirements.py3.Darwin.txt

```diff
@@ -25,7 +25,7 @@ azure-mgmt-batch==7.0.0
 azure-mgmt-batchai==2.0.0
 azure-mgmt-billing==0.2.0
 azure-mgmt-botservice==0.2.0
-azure-mgmt-cdn==3.1.0
+azure-mgmt-cdn==4.0.0
```

Comment:
Please declare version upgrade in release note, too

Reply:
It is already included in the release note.